jsontableschema/types/array.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+from .. import exceptions
+from .. import utilities
+from . import base
+
+
+# Module API
+
+class ArrayType(base.JTSType):
+
+    # Public
+
+    name = 'array'
+    null_values = utilities.NULL_VALUES
+    supported_constraints = [
+        'required',
+        'pattern',
+        'enum',
+        'minLength',
+        'maxLength',
+    ]
+    # ---
+    python_type = list
```

Comment:
There is question. Prev `JTSType.py` and `JTSType.formats` technically was public but have been never mentioned in documentation. So it doesn't look like a part of public API.